src/index.html

```diff
@@ -10,23 +10,54 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Document</title>
+    <title>Airbnb Search Bar</title>
     <link
       rel="stylesheet"
       href="style.css"
```

Comment:
Keep your attributes correctly formatted. The attributes of the link tag should start on new lines with proper indentation to improve readability.